### ISSUE TITLE
Status variables documentation links to dev.mysql.com are invalid

### ISSUE THREAD

**Issue by M393** [phpmyadmin/phpmyadmin#19069]
### Describe the bug

In Status -> Variables clicking the blue question mark icon links to https://dev.mysql.com/doc/refman/5.7/en/open-tables.html
The link should be https://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Open_tables

### To Reproduce

1. Go to Status -> All status variables
2. Click on the question mark in the column Variable

### Expected behavior

Documentation links should be to the correct page.

### Screenshots

![image](https://github.com/phpmyadmin/phpmyadmin/assets/56256405/5810b168-62dd-43a1-af19-dacf8d60aa84)

### Server configuration

- Database version: MySQL 5.7.44
- phpMyAdmin version: 5.2.2-dev

**Comment by M393** [comment]
https://github.com/williamdes/mariadb-mysql-kbs/issues/300 is for MariaDB and they are working, just with additional redirects. 

My links are to dev.mysql.com and no link I tried works. I guess MySQL changed their links, so it should be updated in phpMyAdmin.

**Comment by williamdes** [comment]
> [williamdes/mariadb-mysql-kbs#300](https://github.com/williamdes/mariadb-mysql-kbs/issues/300) is for MariaDB and they are working, just with additional redirects.
> 
> My links are to dev.mysql.com and no link I tried works. I guess MySQL changed their links, so it should be updated in phpMyAdmin.

Hey,
This is my library, it is used for the generation of this link you are reporting.
I think the variable is not yet detected by my library

**Comment by M393** [comment]
All links for status variables I tested are broken. Also `Open_tables` exists in the file you linked:
![image](https://github.com/phpmyadmin/phpmyadmin/assets/56256405/7c9406dd-ab6e-49d8-8b35-e00ce6fd6fd6)

**Comment by williamdes** [comment]
Okay, thanks for this info. I may have missed something. I will test on MySQL 5.7 and fix this

**Comment by williamdes** [comment]
> All links for status variables I tested are broken. Also Open_tables exists in the file you linked:

Seems like they exist but only for MariaDB. So the links are not available on MySQL.
I implemented it for MySQL, I hope this will be okay: https://github.com/williamdes/mariadb-mysql-kbs/commit/0b2a6ad48341fc34aa6136f1fe5b66ddb739a57e

PS: How to you have such variables, Com_xxx stuff is not showing in my list ?

**Comment by M393** [comment]
Looks like all status variables starting with `com_` should link [here](https://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Com_xxx).
See the [status variable reference](https://dev.mysql.com/doc/refman/5.7/en/server-status-variable-reference.html).

Thanks for working on this so quickly. I tested it and the link to `Open_tables` now takes me to the correct location.
Though it links to the MySQL [8.0 kb](https://dev.mysql.com/doc/refman/8.0/en/server-status-variables.html#statvar_Open_tables), not [5.7](https://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Open_tables) (not sure if this is intentional or an oversight).

**Comment by williamdes** [comment]
Thank you for confirming, do you know how you did activate the Com_xxx variables?
On my end here is the list of variables I need to add https://github.com/williamdes/mariadb-mysql-kbs/issues/343

Unfortunately the library does not handle different versions of MySQL in the way where it would send you to the version you are using. It has no knowledge of versions. Only variables.

Did you install the composer dev version?
Or replace with the download of my library in your vendor folder?
Let me know

**Comment by M393** [comment]
I did not do anything to enable these variables, it is just a standard installation of MySQL 5.7.

I change this in composer.json
```patch
-        "williamdes/mariadb-mysql-kbs": "^1.2"
+        "williamdes/mariadb-mysql-kbs": "dev-main"
```

**Comment by williamdes** [comment]
> I did not do anything to enable these variables, it is just a standard installation of MySQL 5.7.

Okay, on the Docker version I do not see them :/

**Comment by M393** [comment]
They are available on the demo server if this helps you:
![image](https://github.com/phpmyadmin/phpmyadmin/assets/56256405/a5fccad1-ebbc-4da6-a42b-f66ecc1a346a)

**Comment by M393** [comment]
Normal status variables work.

Though still getting page not found error for e.g. `Com show open tables`
which links to https://dev.mysql.com/doc/refman/5.7/en/com-show-open-tables.html (404) instead of https://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Com_xxx

**Comment by williamdes** [comment]
> Normal status variables work.
> 
> Though still getting page not found error for e.g. `Com show open tables` which links to https://dev.mysql.com/doc/refman/5.7/en/com-show-open-tables.html (404) instead of https://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Com_xxx

Indeed, they seem to only have been detailed on MariaDB, I need to do some workaround for them
https://github.com/williamdes/mariadb-mysql-kbs/blob/07106dab252127c329cc206cd79cf2f51f989e5e/data/variables/mariadb-server-status-variables.json#L1338
Can you open an issue on my repo ?

**Comment by M393** [comment]
Done: https://github.com/williamdes/mariadb-mysql-kbs/issues/351
